src/pages/components/proposal-list/proposal-list.tsx

```diff
@@ -71,8 +45,44 @@ export default function ProposalList(props: Props) {
   );
 }
 
-export function EmptyState(props: { children: ReactNode }) {
-  return <div className={styles.noProposals}>{props.children}</div>;
+interface ProposalListItemProps {
+  proposal: Proposal;
+  href: string;
+}
+
+function ProposalListItem(props: ProposalListItemProps) {
+  const { proposal, href } = props;
+  const votingSliderData = voteSliderSelector(proposal);
+
+  // Preload proposal creator ENS name
+  useEnsName({ address: proposal.creator as Address });
+
+  return (
```

Comment:
Nothing changed here, just moved this JSX into this `ProposalListItem` component in order to use the `useEnsName()` hook.